docusaurus/docs/dev-docs/plugins/documentation.md

```diff
@@ -263,7 +268,7 @@ module.exports = {
       "x-strapi-config": {
         mutateDocumentation: (generatedDocumentationDraft) => {
           generatedDocumentationDraft.paths[
-            "/answer-to-everything"
+            "/answer-to-everything" // Must be an existing path
```

Comment:
I'm not sure that it does 🤔. Technically you should be able to mutate the object like any other object. Maybe we don't need to specify anything here?